### PR TITLE
Add test verifying es_monitor_type on automatic metrics

### PR DESCRIPTION
**Pull request by jsravn**
Like `up` and `scrape_duration_time`.

Verifies https://github.com/lightbend/helm-charts/pull/178

### CODE REVIEW

File: enterprise-suite/tests/smoke_prometheus [hunk 1]
```diff
@@ -230,6 +232,19 @@ busy_wait app_data
 
 prom_has_data "$app_instances"
 
+# automatic metrics should gain an es_monitor_type label when a custom monitor is created
+curl -XPOST -H "Content-Type: application/json" -H 'Author-Name: Me' -H 'Author-Email: me@lightbend.com' -H 'Message: testing!' \
+    "$ES_MONITOR_API/monitors/es-test/mytest" \
+    --data '{"model":"threshold","parameters":{"name":"my_custom_monitor","metric":"up{es_workload=\"es-test\"}","comparator":"<","threshold":"1","window":"5m","confidence":"0.5","severity":"critical","summary":"pod not up","description":"","monitorType":"es-test"}}'
```
Comment (dbrinegar) [inline enterprise-suite/tests/smoke_prometheus]: shouldn't need `es_workload="es-test"` as a filter.  Scope will handle that.

Comment (dbrinegar) [reply enterprise-suite/tests/smoke_prometheus]: in refactored API, scope will come from `"$ES_MONITOR_API/monitors/es-test/mytest"` and set scope to "es-test" for you, so you also won't need to set monitorType = scope like this. But we need the refactor to land, and then tweak this a little.

**Comment by marcoderama** [comment]
I, uhhh, borrowed heavily from this PR in https://github.com/lightbend/helm-charts/pull/186.  One of us will have some merge conflicts to deal with...  :-)

File: enterprise-suite/tests/smoke_prometheus [hunk 2]
```diff
@@ -19,6 +19,8 @@ set +e
 #
 
 PROM=$( busy_wait nodeport expose-prometheus)
+ES_CONSOLE=$( busy_wait nodeport expose-es-console)
```
[the remark this answers is not in the document]
Comment (jsravn) [reply enterprise-suite/tests/smoke_prometheus]: That's what I get for copy-pasta.

**Pull request merged**